badparking/mobile_api/views.py

```diff
@@ -1,3 +1,37 @@
-from django.shortcuts import render
+from django.http import Http404
+from django.contrib.auth import get_user_model
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
 
-# Create your views here.
+from core.models import CrimeType
+from media.models import MediaFileModel
+from .serializers import ClaimSerializer
+
+
+class ClaimList(APIView):
+    def post(self, request, format=None):
+        serializer = ClaimSerializer(data=request.data)
+
+        if serializer.is_valid():
+            claim = serializer.save()
+            # temporary solution until we'll have auth
+            claim.user = request.user \
```

Comment:
How about using extra kwargs or extra context for passing the user or request to the serializer, then accessing it from `create()` and actually saving there to keep the separation of concerns intact?